src/test/software/amazon/event/ruler/ACMachineTest.java

```diff
@@ -1310,6 +1315,74 @@ public void testAnythingButSuffix() throws Exception {
 
     }
 
+    @Test
+    public void testAnythingButIgnoreCase() throws Exception {
+
+        String rule = "{\n" +
+                "\"a\": [ { \"anything-but\": {\"ignore-case\": [\"yes\", \"please\"]  } } ],\n" +
+                "\"b\": [ { \"anything-but\": {\"ignore-case\": \"no\"  } } ]\n" +
```

Comment:
Why not support this?

Reply:
It does.  both of those syntaxes are valid

Reply:
"a" can be anything but the strings "yes", or "please"
"b" can be anything but the string "no"....  Are my values in the rule confusing?

Reply:
I made an error when reading this earlier in the morning. No need to change. Let me resolve this.